.github/drafts/16506/16506_range_slider_outline.md

```diff
@@ -0,0 +1,76 @@
+*** Range Slider ***
+
+The Range Slider widget allows users to choose a range of values from numerical data.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Search filtering (i.e. Min/ Max price)
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
+
+These properties allow you to edit the Range Slider widget. The following table lists all the widget's unique properties; if you do not see a certain widget property here, it may be listed under [Common Widget Properties]().
+
+| **Property** | **Description** |
+|--------------|-----------------|
+| **Min Value** | The starting/smallest possible value of the slider. The value may be negative and/or include decimals, however it must always be less than the **Max Value**. _(number)_ |
+| **Max Value** | The ending/largest possible value of the slider. The value may be negative and/or include decimals, however it must always be greater than the **Min Value**. _(number)_ |
+| **Step Size** | The increment by which the user can adjust the slider's value. This increment must be at least 0.1, and cannot be less than the **Min Value** or greater than the **Max Value**. _(number)_ |
+| **Min Range** |  |
```

Comment:
**MinRange** property is the range interval between the 2 thumbs of the slider. Say if it is 10 you can only select values `10  & 20` for instance and not `15 & 20`. The interval will always be `10 or greater than it not less than 10`.